01_temperature/temperature.rb

```diff
@@ -0,0 +1,14 @@
+def ftoc(param)
+  if param ===32
+    0
+    else
+       5.0*(param-32)/9
+  end
+end
+
+def ctof (param)
+  if param===0
+    32
+    else 9.0*param/5.0+32
```

Comment:
Do you really need the if ? What happens if you delete it and leave the function do only the else branch? (also reformat the code :) )